panda_connect.py

```diff
@@ -0,0 +1,29 @@
+import asyncio
+from pathlib import Path
+
+import numpy as np
+
+from ophyd_async.core import StaticFilenameProvider, StaticPathProvider
+from ophyd_async.fastcs.panda import HDFPanda, SeqTable
+
+
+async def get_panda():
+    panda = HDFPanda(
+        "PANDA1:", StaticPathProvider(StaticFilenameProvider("test-panda"), Path("."))
+    )
+    await panda.connect()
+    assert isinstance(panda, HDFPanda)
+    table = await panda.seq[1].table.get_value()
+    assert isinstance(table, SeqTable)
+    print("OUTA1", table.outa1)
+    outa2 = table.outa2
+    position = table.position
+    assert isinstance(outa2, np.ndarray) and outa2.dtype == np.bool_, outa2.dtype
```

Comment:
```suggestion
    assert isinstance(outa2, np.ndarray) and outa2.dtype == np.bool_, f"{outa2.dtype} is of wrong type!"
```

Not exactly the wording I want, but I was confused until I realised the arg after the comma was for the AssertionError, not part of the comparison

Reply:
But is this for local testing only and not part of the code to be pushed?